JapaneseFilter/sections/specific.txt

```diff
@@ -3036,6 +3036,10 @@ nc-net.or.jp##.box-pr-image-01
 mori.nc-net.or.jp###ok_ad_side
 gyutte.jp###gt_sp_all_header
 gyutte.jp###gt_sp_all_footer
+japanjournals.com###easycookieinfo
+japanjournals.com##div[class="slidebanner"]
+japanjournals.com##div[class="custom"]
+japanjournals.com##div[class="t3-module module cake-module "]
```

Comment:
It works fine, but maybe something more ad related would be better, for example:
```adblock
japanjournals.com##.t3-sidebar-right .bannergroup
```